src/index.js

```diff
@@ -88,7 +88,7 @@ export function syncReduxAndRouter(history, store, selectRouterState = SELECT_ST
   const unsubscribeHistory = history.listen(location => {
     const route = {
       path: createPath(location),
-      state: location.state
+      state: location.state === null ? undefined : location.state
```

Comment:
hi @kjbekkelund! yes, the tests were failing due to the way history assigns a `null` value to state, so this was the most lightweight way to keep state `undefined` internally